windowsforms/FlexGridShowcaseDemo/cs/MainForm.cs

```diff
@@ -0,0 +1,704 @@
+﻿using C1.Framework;
+using C1.Win.FlexGrid;
+using C1.Win.Themes;
+using C1.Win.Ribbon;
+using C1.Win.RulesManager;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Resources;
+using System.Windows.Forms;
+
+namespace FlexGridShowcaseDemo
+{
+
+    public partial class MainForm : C1RibbonForm 
+    {
+        private const string CustomThemeName = "Greenwich";
+        private const int FooterTextPadding = 4;
+
+        private DataSet _dataSet = new DataSet();
+        private C1RulesManager _rulesManager = new C1RulesManager();
+        private IEnumerable<IRule> _rules = Enumerable.Empty<IRule>();
+        private static Random s_rnd = new Random();
```

Comment:
```suggestion
        private readonly DataSet _dataSet = new DataSet();
        private readonly C1RulesManager _rulesManager = new C1RulesManager();
        private static readonly Random s_rnd = new Random();
```

No need to initialise fields that get assigned before they get used, e.g.
```
        private IEnumerable<IRule> _rules;
```

Reply:
Fixed with 2e37bc2d11fab0b4acc9c6fba633d242eb0a1a1a.